client/src/pages/Profile.js

```diff
@@ -0,0 +1,190 @@
+                                                    <span style={{ fontSize: 14, fontWeight: "bold" }}>Select your Time Zone: </span>
+                                                </Box>
+                                            </Grid>
+                                            <Grid item xs={12} sm={8}>
+                                                <Grid container spacing={0}>
+                                                    <Grid item xs={12} sm={5}>
+                                                        <TextField
+                                                            className={classes.small_Font}
+                                                            variant="outlined"
+                                                            margin="normal"
+                                                            size="small"
+                                                            fullWidth
+                                                            // label="calendapp.com/"
+                                                            placeholder="UTC Time"
+                                                        // onChange={(e) => this.setState({ emailText: e.target.value })}
+                                                        />
+                                                    </Grid>
+                                                    <Grid item xs={12} sm={7}>
+                                                    </Grid>
+                                                </Grid>
+                                            </Grid>
+                                        </Grid>
+                                        <Box className={classes.paper} mt={6}>
+                                            <Button mt={8}
+                                                type="submit"
+                                                variant="contained"
+                                                color="primary"
+                                                className={classes.submit}
+                                            // component={Link} to=""
+                                            // onClick={handleSignUpEmail}
+                                            >
+                                                Continue
+          </Button>
+                                        </Box>
+                                    </Box>
+
+                                </div>
+                            </Container>
+                            <Box mt={1} p={3} pt={2} className={classes.center_Text}>
+                                <span style={{ fontSize: "14px", color: "silver" }}>Set up later
+                            </span>
+                            </Box>
+                        </CardContent>
+                    </Card>
+
+                </div>
+            </div>
+        );
+    }
+}
+
+export default withStyles(useStyles)(Profile);
```

Comment:
i also see a mix of using withStyles and useStyles in different components - i would suggest to stick with one